catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/TreUtility.java

```diff
@@ -41,4 +43,13 @@ public static Serializable getTreValue(TreGroup tre, String key) {
 
         return null;
     }
+
+    static Optional<Integer> findIntValue(Tre tre, String tagName) {
```

Comment:
Why not `public`?

Also, should we be consistent and have both methods return the same thing, `Optional` or `null` but not both? If making them consistent is too much trouble, can we add `@Nullable` on `getTreValue()`?